planning/ThingDescription/td-next-work-items/usability-and-design.md

```diff
@@ -63,12 +63,19 @@ Related Issues:
 
 Also known as: Mapping TD elements to messages
 
-How can we represent all protocol messages sent and received?
-It should be using data schema to indicate application-specific parameters but we should be able to describe payload, header, uri variables, and other protocol parameters.
+- Some protocols have multiple side-channels, e.g HTTP has Headers, URI-Variables, Payload/Body.
```

Comment:
We need to define side-channel. Can we simply say "any means to pass application-relevant data"?

Reply:
Probably it could be worded as "some protocols have a main channel and ancillary channels to deliver information" ?